frontend/apps/home/src/app/grid/components/cell-types/grid-cell-background.ts

```diff
@@ -0,0 +1,23 @@
+import {Component, HostBinding, Input, OnChanges, OnInit} from '@angular/core';
+import {VersionStatus} from '../../services/grid.service';
+
+@Component({
+  selector: 'grid-cell-background',
+  template: `
+    <div class="grid-cell-background">
+      <div class="grid-cell-background-item"></div>
+    </div>
+  `,
+  styleUrls: [
+    'grid-cell-background.scss'
+  ]
+})
+export class GridCellBackground {
```

Comment:
nie zebym sie czepial, ale lubie konwencje, że jeżeli to komponent to plik się zwie - grid-cell-background.component.ts - latwo sie rozeznac w projekcie :)